### DISCUSSION TITLE
How do I make Mojo  🔥 work on replit.com?

### DISCUSSION THREAD

**Discussion by mojo-for-ai** [modular/modular#1400] [General]
My dream is to complete my university studies and learn artificial intelligence, but the only obstacle standing in front of me is my computer. I have a very weak device that is barely fit to run the simplest algorithms. Can someone help me run mojo🔥 on replit.com?

**Comment by manatlan** [comment]
I don't think it could be possible on repl.it ...
but it should be possible on glitch.com

**Comment by lattner** [comment]
We announced on Monday that we'll be working with replit to get first class support for Mojo in replit.  We don't have a committed timeframe though, much integration work to be figured out.

**Reply by Eprahim-taha** [reply]
I think Mojo 🔥 will work on replit.com if Mojo 🔥 reaches version 1.0.

**Reply by mojo-for-ai** [reply]
Isn't Mojo 🔥 supposed to work on replit.com in the upcoming Mojo 🔥 0.7 release?

**Reply by Eprahim-taha** [reply]
It's not that easy.  I think it will take 2-3 months to be able to use Mojo 🔥 within Replit.com.  I mean, I don't think that those responsible for developing Mojo 🔥 and those responsible for the Replit.com website will be able to make Mojo 🔥 0.7 work on the Replit.com website.  It's more complicated than that.  I think you might be able to use Mojo 🔥 inside Replit.com when Mojo 🔥 hits version 1.0, which won't happen for several months, maybe if the Mojo 🔥 development team releases a new version of Mojo 🔥 every month, we're talking about four months for Mojo 🔥 to work  On Replit.com
